course_catalog/serializers.py

```diff
@@ -536,7 +536,6 @@ def create(self, validated_data):
             with transaction.atomic():
                 userlist = super().create(validated_data)
                 userlist.topics.set(CourseTopic.objects.filter(id__in=topics_data))
-                upsert_user_list(userlist)
```

Comment:
Why was this removed? Is it happening elsewhere now?

Reply:
Yes, when a list is created, it has no items so it shouldn't be indexed.  The upsert (or delete) is now occurring when items are added to or removed from the list [here](https://github.com/mitodl/open-discussions/pull/2448/files#diff-be540db5888ee233a285539323d0d9e0R617-R620)